### PR TITLE
Print unexpected entries input file

### LINKED ISSUE #1043: Add driver support to verify input file
Inlet has the ability to list unused symbols/ parameters and we should have a cmd line option to print them in the serac driver

### PR DESCRIPTION
**Pull request by chapman39**
* Adds an option to the serac driver to print unused entries in input files. Fixes #1043 
* Fix lua files with syntax errors (extra comma needed)

### CODE REVIEW

**Comment by chapman39** [comment]
Not much changes but test building docs anyway: https://readthedocs.org/projects/serac/builds/22928776/

File changed: https://serac.readthedocs.io/en/feature-chapman39-print_unexpected_entries_input/sphinx/user_guide/command_line_options.html

**Review by jamiebramwell — APPROVED** [review]

**Comment by jamiebramwell** [comment]
ping @white238

File: src/docs/sphinx/user_guide/command_line_options.rst [hunk 1]
```diff
@@ -41,6 +41,10 @@ Below is the documentation for Serac's command line options:
      - -p
      - N/A
      - Enable ParaView output
+   * - --print-unused-entries-input-file
```
Comment (white238) [inline src/docs/sphinx/user_guide/command_line_options.rst]: Would `--print-unused-input-file-entries` be better? This goes along the same lines as `create-input-file-docs`.

It's also rather lengthy, though very descriptive. Maybe `--print-unused`?

Thoughts @jamiebramwell ?

Comment (jamiebramwell) [reply src/docs/sphinx/user_guide/command_line_options.rst]: I think `--print-unused` is verbose enough.

**Review by white238 — APPROVED** [review]

**Pull request merged**